storage/innobase/xtrabackup/test/t/xb_pause_after_log_status.sh

```diff
@@ -0,0 +1,92 @@
+#
+# 1. start backup and wait a moment after query p_s.log_status
+# 2. insert data and wait for a moment
+# 3. continue backup
+# 4. restore
+# 5. check data
+#
+
+if ! $XB_BIN --help 2>&1 | grep -q debug-sync; then
+    skip_test "Requires --debug-sync support"
+fi
+
+start_server
+
+function backup_local() {
+	xtrabackup --backup --debug-sync=xtrabackup_after_query_log_status --target-dir=$topdir/backup
+}
+
+function prepare_local() {
+	xtrabackup --prepare --target-dir=$topdir/backup
+}
+
+
+function restore() {
+  [[ -d $mysql_datadir ]] && rm -rf $mysql_datadir && mkdir -p $mysql_datadir
+  xtrabackup --copy-back --target-dir=$topdir/backup
+}
+
+function do_test() {
+
+  cat <<EOF | $MYSQL $MYSQL_ARGS
+CREATE DATABASE sakila;
+use sakila;
+CREATE TABLE t (a INT);
+INSERT INTO t (a) VALUES (1), (2), (3);
+EOF
+
+  innodb_wait_for_flush_all
+
+  (eval $backup_cmd) &
+  job_pid=$!
+
+  wait_for_xb_to_suspend $pid_file
+
+  xb_pid=`cat $pid_file`
+
+  checksum1=`checksum_table sakila t`
+  $MYSQL $MYSQL_ARGS -Ns -e "INSERT INTO t (a) VALUES (10), (20), (30);" sakila
```

Comment:
so if you select data from prepared backup dir, 10,20,30 would be rolledback?

Reply:
redo from this insert won't be applied since they have LSN after we queried from log_status.